frontend/packages/operator-lifecycle-manager/src/components/index.tsx

```diff
@@ -114,7 +114,11 @@ export const providedAPIForModel = (csv: ClusterServiceVersionKind, model: K8sKi
 
 export const parseALMExamples = (csv: ClusterServiceVersionKind) => {
   try {
-    return JSON.parse(csv?.metadata?.annotations?.['alm-examples'] ?? '[]');
+    return JSON.parse(
+      csv?.metadata?.annotations?.['operatorframework.io/initialization-resource'] ??
```

Comment:
I have a feeling this is going to break the examples for other resources provided by the operator. Won't this always return the initialization resource sample for any operand?

Reply:
Isn't that the goal here?  If there is an initialization resource sample, then we want to show that over the alm-example since that is what we use to know we need a CR?  I did testing with other non CR CSVs and things looked good, but I will go do some more.

Reply:
@zherman0 No, what I'm saying is if there are multiple provided APIs, this will always use the YAML for the initialization resource for all types instead of the YAML for the correct type. Can you check?